gen/calc.py

```diff
@@ -296,7 +366,18 @@ def calculate_exhibitor_static_ensemble(master_list):
         'ui_banner_header_content': 'null',
         'ui_banner_footer_content': 'null',
         'ui_banner_image_path': 'null',
-        'ui_banner_dismissible': 'null'
+        'ui_banner_dismissible': 'null',
+        'dcos_overlay_network': '{                    \
```

Comment:
"44.128.0.0/16" is a safe set of IPs.
We could choose a different set for VTEPs. Like from 198.18.0.0/15 (maybe 198.19.0/20?)